packages/common/src/api/comments.ts

```diff
@@ -104,9 +155,23 @@ const commentsApi = createApi({
         const sdk = await audiusSdk()
         await sdk.comments.deleteComment(commentData)
       },
-      options: { type: 'mutation' }
+      options: { type: 'mutation' },
+      onQuerySuccess(_res, { id, entityId }, { dispatch }) {
```

Comment:
Deleting after confirmation. TODO: discuss with design if this should be optimistic/pre-confirmation